### DISCUSSION TITLE
Cloudflare Worker prisma/edge not working

### DISCUSSION THREAD

**Discussion by enqqi** [prisma/prisma#19173] [Q&A]
It was running just fine. I updated prisma and reinstalled dependencies then did `npx prisma generate --data-proxy`. Generates the client, but right after starting the server with `wrangler dev src/index.ts` I get 
<img width="687" alt="Screenshot 2023-05-10 at 7 11 57 AM" src="https://github.com/prisma/prisma/assets/13815017/41170c3b-1b39-4463-96a3-1af852228b97">

**Comment by ludralph** [comment]
Hi @osejrafael 👋 

Can you please confirm the version of Prisma you updated to?

**Reply by enqqi** [reply]
Hi @ludralph I updated to `4.14.0`. 
If I manually set `Rp={}` instead of `Rp=require("os")` on `edge.js:10:3383` it works.
Also I tried rolling back to version `4.13.0` and it does not work now. which is weird since the only change I made was update prisma and generate the new client with `npx prisma generate --data-proxy`

**Reply by ludralph** [reply]
Can you please delete your `node_modules` directory and try again?

**Reply by oanogin** [reply]
@osejrafael hi, do you have any updates?

**Reply by enqqi** [reply]
I did that multiple times, even tried with `yarn` and `pnpm`.
Deleted node_modules and lock file then reinstalled everything then generate client.

**Comment by enqqi** [comment]
Opened an issue. Apparently it has something to do with the new `4.14.0` update.
https://github.com/prisma/prisma/issues/19174
